components/Banner/Banner.php

```diff
@@ -72,6 +72,11 @@ public function disableGoogleAnalyticsOnLoad()
                 for (var ccfwCookie = 0; ccfwCookie < ccfwCookiesArrayLength; ccfwCookie++) {
                     document.cookie = ccfwCookies[ccfwCookie] + '=; Path=/; domain=.'+ document.domain
                     +'; Expires=Thu, 01 Jan 1970 00:00:01 GMT;';
+
+                    // Also remove cookies that start with domains without www
+                    var nonWwwDomain = document.domain.slide(3);
```

Comment:
I was thinking this could be: 

```
if(document.domain.includes("www.")){
var nonWwwDomain = s.replace('www.', "")
//do remove
}
```
As its abit more specific. Let me know your thoughts.

Reply:
Ooh I prefer that :) I'll add that in!